### PR TITLE
Fixed auto tests on 1.11.1 and reduced repetitive output when command line args are invalid

### PR DESCRIPTION
**Pull request by barrycaceres**
- Added tracking of last exception logged to reduce repetititve output when command line options are invalid.
- Fixed auto tests when building with version 1.11.1 or later of native Senzing libs
- Updated Changelog and updated to version 1.7.4.

### CODE REVIEW

File: src/main/java/com/senzing/util/LoggingUtilities.java [hunk 1]
```diff
@@ -119,4 +126,59 @@ public static void logError(String      operation,
...
+
+  /**
+   * Checks if the specified {@link Throwable} is the last logged exception.
+   * This is handy for telling if the exception has already been logged by a
+   * deeper level of the stack trace.
+   * @param t The {@link Throwable} to check.
+   * @return <tt>true</tt> if it is the last logged exception, otherwise
+   *         <tt>false</tt>.
+   */
+  public static void setLastLoggedException(Throwable t) {
```
Comment (timadair) [inline src/main/java/com/senzing/util/LoggingUtilities.java]: You repeated your javadoc from isLastLoggedException here.

File: src/main/java/com/senzing/configmgr/ConfigurationManager.java [hunk 2]
```diff
@@ -562,7 +562,7 @@ public static void main(String[] args) throws Exception {
     try {
       options = parseCommandLine(args);
     } catch (Exception e) {
-      e.printStackTrace();
+      if (!isLastLoggedException(e)) e.printStackTrace();
       System.out.println(ConfigurationManager.getUsageString(false));
```
Comment (timadair) [inline src/main/java/com/senzing/configmgr/ConfigurationManager.java]: Were you wanting to include the printing of the usage string in the new if?

File: src/main/java/com/senzing/repomgr/RepositoryManager.java [hunk 3]
```diff
@@ -453,6 +454,7 @@ public static void main(String[] args) throws Exception {
     try {
       options = parseCommandLine(args);
     } catch (Exception e) {
+      if (!isLastLoggedException(e)) e.printStackTrace();
       System.out.println(RepositoryManager.getUsageString(false));
```
Comment (timadair) [inline src/main/java/com/senzing/repomgr/RepositoryManager.java]: Same here.  Were you wanting to include the usage string on repeated exceptions?

File: src/main/java/com/senzing/util/LoggingUtilities.java (hunk 1 above)
Comment (barrycaceres) [reply src/main/java/com/senzing/util/LoggingUtilities.java]: oops

File: src/main/java/com/senzing/configmgr/ConfigurationManager.java (hunk 2 above)
Comment (barrycaceres) [reply src/main/java/com/senzing/configmgr/ConfigurationManager.java]: no -- the problem was that the exception message was appearing twice.... still need the usage string printed regardless

File: src/main/java/com/senzing/repomgr/RepositoryManager.java (hunk 3 above)
Comment (barrycaceres) [reply src/main/java/com/senzing/repomgr/RepositoryManager.java]: same as above -- this is as intended

File: src/main/java/com/senzing/util/LoggingUtilities.java (hunk 1 above)
Comment (barrycaceres) [reply src/main/java/com/senzing/util/LoggingUtilities.java]: I will fix this

**Comment by docktermj** [comment]
Issue number: #80

**Pull request merged**